src/AutosuggestContainer.js

```diff
@@ -69,13 +69,14 @@ export default class AutosuggestContainer extends Component {
     renderSectionTitle: PropTypes.func,
     getSectionSuggestions: PropTypes.func,
     focusInputOnSuggestionClick: PropTypes.bool,
+    selectFirstSuggestion: PropTypes.bool,
     theme: PropTypes.object,
     id: PropTypes.string
   };
 
   static defaultProps = {
     onSuggestionsUpdateRequested: noop,
-    shouldRenderSuggestions: value => value.trim().length > 0,
+    shouldRenderSuggestions: value => value && value.trim().length > 0,
```

Comment:
It's possible for the value to be null (either from default props in the library, or from clients setting the value to null). And if the value is null or undefined, there's no suggestions to render. Does that make sense?

I did just realize that this will return false if value is 0, so I'll need to update this to be more explicit.